app/dashboard/browse/search/organizations/[id]/page.tsx

```diff
@@ -1,9 +1,113 @@
+  const [orgProfiles, setOrgProfiles] = useState([]);
+
+
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const orgData = await getSingleOrg(params.id);
+      if (orgData === null) {
+        toast({
+          variant: "destructive",
+          title: "Unable to retrieve org profile",
+          description: "Reload page or check another time",
+        });
+        setErrorFetching(true);
+        return;
+      }
+      setOrgName(orgData.name);
+      setEmail(orgData.email);
+      setBio(orgData.bio);
+      setWebsite(orgData.website);
+      setOrgType(orgData.isNonProfit);
+
+      const orgProfilesData = await getOrgProfiles(params.id);
+      if (orgProfilesData === null) {
+        toast({
+          variant: "destructive",
+          title: "Unable to retrieve team profiles",
+          description: "Reload page or check another time",
+        });
+        setErrorFetching(true);
+        return;
+      }
+      setOrgProfiles(orgProfilesData);
+
+    }
+
+    fetchData();
+  }, []);
+
+  function Card({ id, name, email }: { id: string, name: string, email: string }) {
+    return (
+      <Link
+        href={`/dashboard/browse/search/${id}`}
+        className="border-2 border-neutral-400 bg-[#FFFEFE] 2xl:w-72 w-64 py-3 2xl:px-5 px-4 2xl:mr-5 mr-4 2xl:mb-4 mb-3 rounded-lg group hover:text-darkmaroon hover:border-darkmaroon transition duration-500"
+      >
+        <div className="font-semibold truncate">{name}</div>
+        <div className="text-sm text-body truncate group-hover:text-darkmaroon transition duration-500">
+          {email}
+        </div>
+      </Link>
+    );
+  }
 
-export default function OrganizationPage({ params }: { params: { id: string } }) {
   return (
     <div className="">
       <PageHeading header="Search" description="Organization profile" />
+      {errorFetching ? (
+        <div className="mt-8 text-center">
+          There was an error retrieving organization profile. Reload or try again later.
+        </div>
+      ) : (
+        <div className="mt-8">
+          <div className="2xl:w-[50vw] w-[50vw] 2xl:h-[65vh] h-[55vh] 2xl:px-10 px-6 2xl:py-8 py-6 mb-4 bg-[url('/assets/backgrounds/section_background.svg')] bg-no-repeat bg-cover bg-bottom rounded-3xl shadow-inner shadow-xl">
+            <div className="2xl:text-5xl text-4xl 2xl:ml-7 ml-5 2xl:mt-7 mt-5 2xl:mb-2 mb-2 font-semibold text-darkmaroon">{orgName}</div>
+            <div className="2xl:text-2xl text-2xl 2xl:ml-7 ml-5 2xl:mb-2 mb-2 ">{email}</div>
+            <div className="2xl:text-xl text-2xl 2xl:ml-7 ml-5 2xl:mb-10 mb-6 text-gray-500 opacity-60">{website}</div>
+            <div className="2xl:w-96 w-80 py-3 2xl:ml-7 ml-5 2xl:text-lg text-lg 2xl:mb-12 mb-4 ">{bio}</div>
+            <div className="2xl:text-xl text-xl 2xl:ml-7 ml-5 2xl:mb-2 mb-2 font-medium">Org Type</div>
+            <div className="border-b border-body 2xl:w-72 2xl:ml-7 ml-5 2xl:mb-px mb-px"></div>
```

Comment:
You can put this border on the org type div so it doesn't need an empty div just for border.
<img width="1468" alt="Screenshot 2024-04-28 at 7 38 35 PM" src="https://github.com/nam-t24/Senior-CS-Project/assets/71054634/c6b8694f-eabc-4f99-ba69-29e805358195">
Border also too long for smaller screens

Reply:
Advise on controlling border padding from text? Tried using border-spacing-y with no results

Reply:
add 2xl:w-72 w-64 border-b border-body pb-4 to the org type text div